core/src/main/java/jenkins/model/experimentalflags/RemovePrototypeUserExperimentalFlag.java

```diff
@@ -38,7 +38,7 @@ public RemovePrototypeUserExperimentalFlag() {
     @NonNull
     @Override
     public Boolean getDefaultValue() {
-        return false;
+        return true;
```

Comment:
Rather than changing the default value of the user experimental flag, I think we should completely remove Prototype, including:

- Removing the user experimental flag
- Removing `prototype.js` from the source tree
- Removing all calls to `Object.toJSON`
- Removing `Form.findMatchingInput`
- Removing the customization to `Ajax.Request.prototype.dispatchException`
- Removing `jenkins.stringify`
- Removing `ExceptionTest`

If we fail to do all of the above, we will end up in a state where we have to support both Prototype and non-Prototype in the long term. I believe it is more desirable to explicitly drop any support for Prototype to reduce long term maintenance cost.